src/frontEnd/index.jsx

```diff
@@ -1,21 +0,0 @@
-import React from 'react';
-import reactDOM from 'react-dom';
-import store from './store.js';
-import {Provider} from 'react-redux';
-import App from './components/containers/App.js';
-import Navbar from './components/containers/Navbar.js';
-const Main = () => {
-  return (
-    <Provider store={store}>
-      <div>
-        <Navbar />
-        <div className="container">
-          <App/>
```

Comment:
Overall very clean, well-written code. Please add more details on either the index, app page, or some sort of index/home page that gives more description about how to use your site and how it works. Also, consider adding more stylistic elements, like hover states for the room calendars. Also, please add a demo account so those without a mercycorps / gsg account can at least see functionality.